mobile/src/main/java/org/hawkular/client/android/explorer/InventoryExplorerActivity.java

```diff
@@ -65,14 +68,18 @@
     private TreeNode.BaseNodeViewHolder holder;
     private TreeNode root;
     private Callback<String> callback;
+    Toolbar toolbar;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_inventory_explorer);
+        toolbar = (Toolbar) findViewById(R.id.toolbar);
```

Comment:
@pg301 Hawkular use ButterKnife for binding UI

Can you please visit  [sample](https://github.com/hawkular/hawkular-android-client/blob/master/mobile/src/main/java/org/hawkular/client/android/fragment/AlertsFragment.java) to have clearer understanding

Happy Open Source